ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecureAPI.kt

```diff
@@ -36,12 +37,16 @@ internal class ThreeDSecureAPI(
     }
 
     fun authenticateCardinalJWT(
-        threeDSecureParams: ThreeDSecureParams,
-        cardinalJWT: String,
+        threeDSecureParams: ThreeDSecureParams?,
+        cardinalJWT: String?,
         callback: ThreeDSecureResultCallback
     ) {
-        val lookupCardNonce = threeDSecureParams.threeDSecureNonce
-        val lookupNonce = threeDSecureParams.threeDSecureNonce?.string
+        if (threeDSecureParams == null || cardinalJWT == null) {
+            callback.onThreeDSecureResult(null, BraintreeException("An unexpected error occurred"))
```

Comment:
Do we want to reuse the previous error "threeDSecureParams or jwt is null" here?

Reply:
I don't know that felt like leaking internal implementation details - the merchant shouldn't know about that params or JWT object. What do you think?

Reply:
Ah ok - put it back to the params/jwt null error message for troubleshooting!